zipkin-storage/zipkin2_cassandra/README.md

```diff
@@ -66,6 +66,13 @@ Note: annotations with values longer than 256 characters
 are not written to the `annotation_query` SASI, as they aren't intended
 for use in user queries.
 
+The `trace_by_service_span` index is only used by query apis, and notably supports centisecond
+resolution duration. In other words, query inputs are rounded up to the next 10 milliseconds. For
+example, a call to GET /api/v2/traces?minDuration=12345 will returns traces who include a span that
+has at least 20 milliseconds (20000 microsecond) duration. This resolution only affects the query:
+original duration data remains at microsecond granularity. Meanwhile, write performance is
+dramatically better than writing discrete values, via fewer distinct writes.
+
```

Comment:
this isn't true anymore.
(it was added and then later removed)

after further benchmarking it was seen that rounding from millisecond to centisecond was not a good idea.
(it is still rounded from microsecond to millisecond)

(this whole commit e268af9 can just be removed)